### ISSUE TITLE
ImagePickerFragment CameraPicker

### ISSUE THREAD

**Issue by NoelChew** [coomar2841/android-multipicker-library#2]
Referring to ImagePickerFragment:
```
public void takePicture() {
        cameraPicker = new CameraImagePicker(this);
        cameraPicker.shouldGenerateMetadata(true);
        cameraPicker.shouldGenerateThumbnails(true);
        pickerPath = cameraPicker.pickImage();
    }
```
Missing 
`cameraPicker.setImagePickerCallback(this);`

**Comment by coomar2841** [comment]
Thanks, will fix that.

**Comment by coomar2841** [comment]
Done. Will be available in v0.9.5
